components/CreateEventForm/CreateEventForm.js

```diff
@@ -32,29 +53,51 @@ export default function CreateEventForm(props) {
     }
   }
 
+  const onChange = (event, selectedDate) => {
+    const currentDate = selectedDate || date;
+    setShow(Platform.OS === 'ios');
+    setDate(currentDate);
+  };
+
   return (
     <View style={styles.container}>
-      <Text style={styles.text}>CREATE A NEW EVENT</Text>
-
-      <View style={styles.dropdownContainer}>
-        <Text style={styles.text}>Name of Activity:</Text>
-        <ModalDropdown
-          renderButtonText={nameOfActivity => setNameOfActivity(nameOfActivity)}
-          textStyle={styles.dropdownText}
-          dropdownTextStyle={styles.dropdownTextStyle}
-          defaultValue='>> select activity <<'
-          options={['Volleyball', 'Spikeball', 'Baseball', 'Basketball', 'Horseshoes', 'Beer Olympics']}
-        />
+
+      <View style={styles.inputContainer}>
+        <Text style={styles.titleText}>CREATE  EVENT</Text>
+        <Text style={styles.text}>Select Date:</Text>
       </View>
 
-      <TextInput
-        style={styles.input}
-        placeholder='Location...'
-        onChangeText={location => setLocation(location)}
-        value={location}
+      <DateTimePicker
+        style={styles.picker}
+        testID="dateTimePicker"
+        value={date}
+        mode={mode}
+        display="calender"
+        onChange={onChange}
       />
 
-      <TextInput style={styles.input} placeholder='Date...(NEEDS DATEPICKER)'/>
+      <View style={styles.timeContainer}>
+        <View style={styles.dropdownContainer}>
+          <Text style={styles.text}>Activity:</Text>
+          <ModalDropdown
+            renderButtonText={nameOfActivity => setNameOfActivity(nameOfActivity)}
+            textStyle={styles.dropdownText}
+            dropdownTextStyle={styles.dropdownTextStyle}
+            defaultValue='>> select <<'
+            options={['Baseball', 'Basketball', 'Biking', 'Bocce ball', 'Climbing', 'Croquet', 'Disc golf', 'Fishing', 'Football', 'Hand ball', 'Kickball', 'Outdoor exercising', 'Slacklining', 'Soccer', 'Spike ball', 'Tennis', 'Trail hiking/walking', 'Ultimate frisbee', 'Volleyball']}
```

Comment:
Only thing that might be a stretch goal is a 'custom' activity where they can enter the name of a one-off activity they want for the event.